arroyo/examples/transform_and_produce/script.py

```diff
@@ -0,0 +1,53 @@
+from arroyo.backends.kafka.configuration import build_kafka_consumer_configuration
+from arroyo.backends.kafka.consumer import (
+    KafkaConsumer,
+    KafkaProducer,
+)
+from arroyo.processing.processor import StreamProcessor
+
+
+from arroyo.types import Topic
+import socket
+
+from arroyo.examples.transform_and_produce.factory import (
+    HashPasswordAndProduceStrategyFactory,
+)
+
+
+RAW_TOPIC = Topic("raw-topic")
+HASHED_TOPIC = Topic("hashed-topic")
+
+
+if __name__ == "__main__":
+
+    # Create a Producer, KafkaProducer is a wrapper on confluent_kafka.Producer
+    producer = KafkaProducer(
+        {
+            "bootstrap.servers": "localhost:9092",  # Where the broker is, localhost:9092 when running locally
+            "client.id": socket.gethostname(),
+        }
+    )
+
+    # KafkaConsumer is a wrapper on confluent_kafka.Consumer
+    consumer = KafkaConsumer(
+        # A function to bootstrap required configs to create a consumer
+        build_kafka_consumer_configuration(
+            default_config={
+                "bootstrap.servers": "localhost:9092",
+            },
+            auto_offset_reset="latest",  # "define the behavior of the consumer when there is no committed position" - confluent docs
```

Comment:
I'd remove this comment, auto_offset_reset means a slightly different thing in Arroyo than confluent kafka